framework/CHANGELOG.md

```diff
@@ -4,16 +4,9 @@ Yii Framework 2 Change Log
 2.0.10 under development
 ------------------------
 
-- Bug #7670: Added `UrlNormalizer` for normalizing requests with and without trailing slashes (rob006, cronfy, klimov-paul)
+- Bug #7670: Added `yii\web\UrlNormalizer` for normalizing requests with and without trailing slashes (rob006, cronfy, klimov-paul)
 - Bug #9027: Fixed descendant class of `yii\web\UploadedFile` returns parent instances in case invoked after it (andrewnester)
-- Bug #12428: Fixed `yii\db\mysql\QueryBuilder` causes warning when insert default rows into a table without primary key (DrmagicE)
-- Enh #6996: Added `yii\web\MultipartFormDataParser`, which allows proper processing of 'multipart/form-data' encoded non POST requests (klimov-paul)
-- Enh #9989: ActiveForm now respects formtarget, formmethod and formenctype attributes of submit button (AnatolyRugalev)
-- Enh #12296: Added value validation to `yii\log\Target::setLevels()` (Mak-Di)
-- Enh #12193: Added the ability to suppress the generation of duplicate error messages in `Html::errorSummary`. Added the ability to display error messages beyond the first error for each model attribute (PowerGamer1)
-- Enh #12073: Added the ability to suppress the generation of input hint when it is specified through  `Model::attributeHints()` (PowerGamer1)
-- Bug #7670: Added `UrlNormalizer` for normalizing requests with and without trailing slashes (rob006, cronfy, klimov-paul)
-- Bug #9101: Fixed `yii\web\View` to respect `yii\web\AssetManager::appendTimstamp` property (githubjeka, silverfire)
+- Bug #9101: Fixed `yii\web\View` to respect `yii\web\AssetManager::$appendTimestamp` property (githubjeka, silverfire)
```

Comment:
`yii\web\AssetManager::appendTimestamp` meas that `AssetManager` has `appendTimestamp` **contant**. You can have constant and property that share the same name, but you can't have static and non static property with the same name. So using `$` for properties is more explicit. It also follow PHP convention: 
![6b048660](https://cloud.githubusercontent.com/assets/5972388/19078096/4f48095c-8a4f-11e6-83a0-d6425db9be07.png)